$NetBSD$

Build the panel applets with Gtk+ 3.0
The "embed" panel applet requires an additional library with Gtk+ 3.0

--- src/applets/Makefile.orig	2013-06-11 21:40:17.000000000 +0000
+++ src/applets/Makefile
@@ -5,9 +5,9 @@ LIBDIR	= $(PREFIX)/lib
 CC	= cc
 CPPFLAGSF= -I ../../include
 CPPFLAGS=
-CFLAGSF	= -W `pkg-config --cflags libSystem gtk+-2.0` -fPIC
+CFLAGSF	= -W `pkg-config --cflags libSystem gtk+-3.0` -fPIC
 CFLAGS	= -Wall -g -O2 -pedantic
-LDFLAGSF= `pkg-config --libs libSystem gtk+-2.0`
+LDFLAGSF= `pkg-config --libs libSystem gtk+-3.0`
 AR	= ar
 RANLIB	= ranlib
 CCSHARED= $(CC) -shared
@@ -69,8 +69,8 @@ desktop.so: $(desktop_OBJS)
 	$(CCSHARED) -o desktop.so $(desktop_OBJS) $(desktop_LDFLAGS)
 
 embed_OBJS = embed.o
-embed_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) $(CFLAGSF) $(CFLAGS) `pkg-config --cflags libDesktop`
-embed_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs libDesktop`
+embed_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) $(CFLAGSF) $(CFLAGS) `pkg-config --cflags libDesktop gtk+-x11-3.0`
+embed_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs libDesktop gtk+-x11-3.0`
 
 embed.so: $(embed_OBJS)
 	$(CCSHARED) -o embed.so $(embed_OBJS) $(embed_LDFLAGS)
